src/analyses/variable-sensitivity/variable_sensitivity_dependence_graph.cpp

```diff
@@ -438,20 +438,18 @@ bool variable_sensitivity_dependence_domaint::merge(
 }
 
 /**
- * Perform a context aware merge of the changes that have been applied
- * between function_start and the current state. Anything that has not been
- * modified will be taken from the \p function_call domain.
- *
- * \param function_call: The local of the merge - values from here will be
- *   taken if they have not been modified
- * \param function_start: The base of the merge - changes that have been made
+ * Merges just the things that have changes between
+ * "function_start" and "function_end" into "this".
+ * To be used correctly "this" must be derived from the function
+ * call site.  Anything that is not modified in the function (such
+ * as globals) will not be changed.
+ * \param function_start: The base of the diff - changes that have been made
  *   between here and the end will be retained.
  * \param function_end: The end of the merge - changes that have been made
-///   between the start and here will be retained.
+ *   between the start and here will be retained.
  * \param ns: The global namespace
  */
```

Comment:
Should perhaps use `\copydoc variable_sensitivity_domaint::merge_three_way_function_return`.

Reply:
Done!